docs/udf_generation.md

```diff
@@ -23,3 +25,117 @@ spark.sql("select brand as brand, previous_years_sales(brand, us_sales, sales_ch
 | Honda         |   1315225|
 | Hyundai       |    739045|
 
+## Example 2: Parse heterogeneous JSON text
+
+```python
+from typing import List
+
+@spark_ai.udf
+def parse_heterogeneous_json(json_str: str, schema: List[str]) -> List[str]:
+    """Extract fields from heterogeneous JSON string based on given schema in a right order.
+    If field is missing replace it by None. All imports should be inside function."""
+    ...
+```
+
+`df.show()`
```

Comment:
Hi @SemyonSinchenko , the doc is now available at https://pyspark.ai/udf_generation/
Shall we show how `df` is created instead?

Reply:
Or, let's use `df.show(False)` which will not truncate the output

Reply:
> Hi @SemyonSinchenko , the doc is now available at https://pyspark.ai/udf_generation/
> Shall we show how `df` is created instead?

I'll extend docs snd also change the view of results.